hooks/build

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+set -xeu
+set -o pipefail
+
+AGORA_VERSION=HEAD
+
+GIT_DESCRIBE_OUT=`git describe`
+
+if [ -n "$GIT_DESCRIBE_OUT"]
+then
+	AGORA_VERSION=$GIT_DESCRIBE_OUT
+fi
+
+echo "Agora version is: $AGORA_VERSION"
+
+docker build --build-arg AGORA_VERSION=$AGORA_VERSION -f $DOCKERFILE_PATH -t $IMAGE_NAME .
```

Comment:
I guess all those variables are provided by docker hub ?

Reply:
yes, I think if we didn't have the build file, then the command that is executed is:
`docker build -f $DOCKERFILE_PATH -t $IMAGE_NAME .`